apps/voting/app/src/components/Vote.js

```diff
@@ -102,14 +102,14 @@ function Vote({ vote, onVote, onExecute }) {
               margin-top: ${2.5 * GU}px;
             `}
           >
-            <div
+            <h1
```

Comment:
Could we also use h3s or etc. for things like the `DESCRIPTION`, `CREATED BY`, etc. headings?

Reply:
Done in 56cda30.